infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -40,6 +43,12 @@ export class AtatWebApiStack extends cdk.Stack {
     // Ideally we'd define different stages for dev, test, and staging. For now, a single
     // stage for everything being dev is good enough for a proof of concept
     const restApi = new apigw.RestApi(this, "AtatWebApi", {
+      binaryMediaTypes: ["multipart/form-data"],
+      deployOptions: {
```

Comment:
Are we ready for this?

Reply:
I enabled it to help with my own development but happy to leave it out for now